.github/workflows/ci-latest-helm-chart-release.yaml

```
@@ -10,7 +10,14 @@ jobs:
   publish-chart:
     name: Update Stable Helm Chart With Latest Changes
     if: ${{ (github.repository == 'kubearmor/kubearmor') && (!contains(github.event.head_commit.message, '[skip ci]')) }}
-    runs-on: ubuntu-20.04
+    runs-on: ${{ matrix.os }}
+    env:
+      RUNTIME: ${{ matrix.runtime }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-22.04]
+        runtime: ["docker", "containerd", "crio"]
```

```suggestion
```

We don't runtime matrix for anything other than the test where we already have it. Please don't add runntime variables or fail fast variables. It will affect the flow of the release.

---

Please signoff your commit as well. Check DCO Action failure logs.

---

Hello @daemon1024, thanks for reviewing. I have made the changes that you asked for, runtime and fail fast is removed from wherever it was not earlier there in the workflow. Also I have signed off my commit.